nautobot/ui/src/components/core/BSTable.jsx

```diff
@@ -1,40 +1,40 @@
-import { Form, Table } from "react-bootstrap"
+import { Checkbox, Table, Thead, Tr, Th, Tbody, Td } from "@chakra-ui/react"
```

Comment:
Note that there's currently an open PR against `nautobot-ui` to add styled Checkbox and Table components. Once that's merged we can update to pull from there.

Reply:
Merged now, FYI.